test/dds/core/xtypes/myTest.cpp

```diff
@@ -473,83 +478,85 @@ DynamicData create_dynamic_data(long double pi, StructType& the_struct/*, Struct
-				the_data["bool"].value(true) ;
-    the_data["uint8_t"].value(uint8_t(230)) ;
-				the_data["int16_t"].value<>(int16_t(-784));
-				the_data["uint16_t"].value<>(uint16_t(784));
-				the_data["int32_t"].value<>(int32_t(-5469372));
-				the_data["uint32_t"].value<>(uint32_t(45350234)) ;
-				the_data["int64_t"].value<>(int64_t(-1234523556));
-				the_data["uint64_t"].value<>(uint64_t(1234523556));
-				the_data["float"].value<>(float(3.1415926)) ;
-				the_data["double"].value<>(double(-3.14159264));
-				
-				the_data["long_double"].value<>(pi);
-
-    for(int i = 0 ; i < 1E4 ; ++i) // creating "sequence"
-				{
-				    DynamicData tmp_data(inner_struct);
-								tmp_data["inner_string"].string("lay_down_and_cry") ;
-								tmp_data["inner_float"].value(float(3.1415)) ;
-        for (int j = 0 ; j < 1E3 ; ++j) // creating "sequence.inner_sequence_string"
-								{
-								    StringType s ;
-								    DynamicData dst(s) ;
-												dst.string("another_prick_in_the_world");
-								    tmp_data["inner_sequence_string"].push(dst) ;
-								}
-								
-								for (int j = 0 ; j < 1E2 ; ++j) // creating "sequence.inner_sequence_struct"
-								{
-								    DynamicData tmp_inner_data(second_inner_struct);
-												tmp_inner_data["second_inner_string"].string("paint_it_black");
-												tmp_inner_data["second_inner_uint32_t"].value<>(uint32_t(38)) ;
-												for(int k = 0 ; k < 1E1 ; ++k) //creating "sequence.inner_sequence_struct.second_inner_array"
-												{
-												    tmp_inner_data["second_inner_array"][k].value<uint8_t>(56) ;
-												}
-								    tmp_data["inner_sequence_struct"].push(tmp_inner_data);
-								}
-								for(int j = 0 ; j < 1E1 ; ++j)
-								{
-								    for(int k = 0 ; k < 1E1 ; ++k)
-												{
-                the_data["array"][j][k].value<long double>(3.1415) ;
-												}
-								}
+        Member("array", ArrayType(ArrayType(primitive_type<long double>(), 10), 10))).add_member(
+        Member("sequence", SequenceType(inner_struct)));
+                
+    DynamicData the_data(the_struct);
+    the_data["bool"].value(true);
+    the_data["uint8_t"].value(uint8_t(230));
+    the_data["int16_t"].value<>(int16_t(-784));
+    the_data["uint16_t"].value<>(uint16_t(784));
+    the_data["int32_t"].value<>(int32_t(-5469372));
+    the_data["uint32_t"].value<>(uint32_t(45350234));
+    the_data["int64_t"].value<>(int64_t(-1234523556));
+    the_data["uint64_t"].value<>(uint64_t(1234523556));
+    the_data["float"].value<>(float(3.1415926));
+    the_data["double"].value<>(double(-3.14159264));
+                
+    the_data["long_double"].value<>(pi);
+
+    for(int i = 0; i < 1E1; ++i) // creating "sequence"
+    {
+        DynamicData tmp_data(inner_struct);
+        tmp_data["inner_string"].string("lay_down_and_cry");
+        tmp_data["inner_float"].value(float(3.1415));
+        for (int j = 0; j < 1E1; ++j) // creating "sequence.inner_sequence_string"
```

Comment:
Store these values in a define/var, so it can be changed easily.

Reply:
I'm taking the assumption that this should not be a case.
if u wanna change the struct of a single structure, is much more useful changing the for() line than changing some define value at the beginning of the file